doc/sphinx/addendum/canonical-structures.rst

```diff
@@ -396,7 +396,7 @@ instances of the ``LEQ`` class.
 
   Definition packager T e0 le0 (m0 : LEQ.mixin e0 le0) :=
     [find e | EQ.obj e ~ T | "is not an EQ.type" ]
-    [find o | LE.obj o ~ T | "is not an LE.type" ]
+    [find o | LE.obj o ~ T | "is not a LE.type" ]
```

Comment:
I pronounce `LE` as `el-iː`, so it seems that `an` would be correct the correct option, unless there is some language rule I don't know about.